### PR TITLE
Fix weathergov api precipitationLastHour 

### PR DESCRIPTION
**Pull request by JasonStieber**
Pull request fixes small big in weathergov api format mismatch #3124

### CODE REVIEW

File: modules/default/weather/providers/weathergov.js [hunk 1]
```diff
@@ -210,7 +210,7 @@ WeatherProvider.register("weathergov", {
 		currentWeather.minTemperature = currentWeatherData.minTemperatureLast24Hours.value;
 		currentWeather.maxTemperature = currentWeatherData.maxTemperatureLast24Hours.value;
 		currentWeather.humidity = Math.round(currentWeatherData.relativeHumidity.value);
-		currentWeather.precipitationAmount = currentWeatherData.precipitationLastHour.value;
+		currentWeather.precipitationAmount = currentWeatherData.precipitationLast3Hours.value;
```
Comment (rejas) [inline modules/default/weather/providers/weathergov.js:213]: Maybe use the 3hours info as a fallback in case the precipitationLastHour is undefined?

**Review by rejas — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -68,13 +68,11 @@
...
 		"suncalc": "^1.9.0"
 	},
-	"optionalDependencies": {
-		"electron": "^25.1.0"
-	},
 	"dependencies": {
 		"colors": "^1.4.0",
 		"console-stamp": "^3.1.1",
 		"digest-fetch": "^2.0.3",
+		"electron": "^25.2.0",
```
Comment (rejas) [inline package.json:75]: electron should stay in the optional dependencies

**Comment by khassel** [comment]
@JasonStieber can you make the requested changes?

**Comment by rejas** [comment]
pinging @JasonStieber in the hope he picks this PR up again

**Comment by khassel** [comment]
Looking into the data getting from the weather gov api there is still a property `precipitationLastHour`:

grafik

@rejas So we could implement a logical change to use `precipitationLast3Hours` if `precipitationLastHour` is `null` or close this PR.

**Comment by rejas** [comment]
> @rejas So we could implement a logical change to use `precipitationLast3Hours` if `precipitationLastHour` is `null` or close this PR.

I would vote for the logical change. And you and @sdetweil ?

**Comment by khassel** [comment]
> I would vote for the logical change.

I implemented this now.

**Review by rejas — APPROVED** [review]

**Pull request merged**